server/core/runtime/run_step_runner_test.go

```diff
@@ -111,9 +108,8 @@ func TestRunStepRunner_Run(t *testing.T) {
 			DefaultTFVersion:  defaultVersion,
 			TerraformBinDir:   "/bin/dir",
 		}
-		t.Run(c.Command, func(t *testing.T) {
-			tmpDir, cleanup := TempDir(t)
-			defer cleanup()
```

Comment:
why are we removing the cleanup function here?

Reply:
t.TempDir() automatically cleans stuff up.